The job `mingw-check` of your PR [failed](https://dev.azure.com/rust-lang/rust/_build/results?buildId=18441&view=logs&jobId=74054d28-c774-5fab-ecc2-352fe71b230e) ([pretty log](https://rust-lang.github.io/rust-log-analyzer/log-viewer/#https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_apis/build/builds/18441/logs/66), [raw log](https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_apis/build/builds/18441/logs/66)). Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.

<details><summary><i>Click to expand the log.</i></summary>

```plain
2020-01-14T21:48:23.5765249Z ##[command]git remote add origin https://github.com/rust-lang/rust
2020-01-14T21:48:23.5862894Z ##[command]git config gc.auto 0
2020-01-14T21:48:23.6124815Z ##[command]git config --get-all http.https://github.com/rust-lang/rust.extraheader
2020-01-14T21:48:23.6198665Z ##[command]git config --get-all http.proxy
2020-01-14T21:48:23.6353326Z ##[command]git -c http.extraheader="AUTHORIZATION: basic ***" fetch --force --tags --prune --progress --no-recurse-submodules --depth=2 origin +refs/heads/*:refs/remotes/origin/* +refs/pull/68234/merge:refs/remotes/pull/68234/merge
---
2020-01-14T21:53:12.4201121Z   local time: Tue Jan 14 21:53:12 UTC 2020
2020-01-14T21:53:12.6971069Z   network time: Tue, 14 Jan 2020 21:53:12 GMT
2020-01-14T21:53:12.6975506Z == end clock drift check ==
2020-01-14T21:53:13.6312567Z 
2020-01-14T21:53:13.6419152Z ##[error]Bash exited with code '1'.
2020-01-14T21:53:13.6450036Z ##[section]Starting: Checkout
2020-01-14T21:53:13.6452827Z ==============================================================================
2020-01-14T21:53:13.6452883Z Task         : Get sources
2020-01-14T21:53:13.6452928Z Description  : Get sources from a repository. Supports Git, TfsVC, and SVN repositories.
```

</details><p></p>

[I'm a bot](https://github.com/rust-ops/rust-log-analyzer)! I can only do what humans tell me to, so if this was not helpful or you have suggestions for improvements, please ping or otherwise contact **`@TimNN`**. ([Feature Requests](https://github.com/rust-ops/rust-log-analyzer/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request))

--- a/src/liballoc/lib.rs
+++ b/src/liballoc/lib.rs
@@ -104,7 +104,6 @@
 #![feature(ptr_offset_from)]
 #![feature(rustc_attrs)]
 #![feature(receiver_trait)]
-#![feature(slice_from_raw_parts)]
 #![feature(specialization)]
 #![feature(staged_api)]
 #![feature(std_internals)]

--- a/src/libcore/ptr/mod.rs
+++ b/src/libcore/ptr/mod.rs
@@ -252,7 +252,7 @@ pub(crate) struct FatPtr<T> {
 ///
 /// // create a slice pointer when starting out with a pointer to the first element
 /// let x = [5, 6, 7];
-/// let ptr = &x[0] as *const _;
+/// let ptr = x.as_ptr();
 /// let slice = ptr::slice_from_raw_parts(ptr, 3);
 /// assert_eq!(unsafe { &*slice }[2], 7);
 /// ```

--- a/src/libcore/ptr/mod.rs
+++ b/src/libcore/ptr/mod.rs
@@ -248,17 +248,16 @@ pub(crate) struct FatPtr<T> {
 /// # Examples
 ///
 /// ```rust
-/// #![feature(slice_from_raw_parts)]
 /// use std::ptr;
 ///
 /// // create a slice pointer when starting out with a pointer to the first element
-/// let mut x = [5, 6, 7];
-/// let ptr = &mut x[0] as *mut _;
-/// let slice = ptr::slice_from_raw_parts_mut(ptr, 3);
+/// let x = [5, 6, 7];
+/// let ptr = &x[0] as *const _;
+/// let slice = ptr::slice_from_raw_parts(ptr, 3);
 /// assert_eq!(unsafe { &*slice }[2], 7);
 /// ```
 #[inline]
-#[unstable(feature = "slice_from_raw_parts", reason = "recently added", issue = "36925")]
+#[stable(feature = "slice_from_raw_parts", since = "1.42.0")]
 #[rustc_const_unstable(feature = "const_slice_from_raw_parts", issue = "67456")]
 pub const fn slice_from_raw_parts<T>(data: *const T, len: usize) -> *const [T] {
     unsafe { Repr { raw: FatPtr { data, len } }.rust }
@@ -275,7 +274,7 @@ pub const fn slice_from_raw_parts<T>(data: *const T, len: usize) -> *const [T] {
 /// [`slice_from_raw_parts`]: fn.slice_from_raw_parts.html
 /// [`from_raw_parts_mut`]: ../../std/slice/fn.from_raw_parts_mut.html
 #[inline]
-#[unstable(feature = "slice_from_raw_parts", reason = "recently added", issue = "36925")]
+#[stable(feature = "slice_from_raw_parts", since = "1.42.0")]
 #[rustc_const_unstable(feature = "const_slice_from_raw_parts", issue = "67456")]
 pub const fn slice_from_raw_parts_mut<T>(data: *mut T, len: usize) -> *mut [T] {
     unsafe { Repr { raw: FatPtr { data, len } }.rust_mut }

--- a/src/libcore/tests/lib.rs
+++ b/src/libcore/tests/lib.rs
@@ -37,7 +37,6 @@
 #![feature(iter_is_partitioned)]
 #![feature(iter_order_by)]
 #![feature(cmp_min_max_by)]
-#![feature(slice_from_raw_parts)]
 #![feature(const_slice_from_raw_parts)]
 #![feature(const_raw_ptr_deref)]
 

